lucet-runtime/tests/instruction_counting.rs

```diff
@@ -33,7 +35,41 @@ pub fn check_instruction_counts() {
             );
             ent
         })
-        .collect();
+        .collect()
+}
+
+#[test]
+pub fn check_instruction_count_off() {
+    let files: Vec<DirEntry> = get_instruction_count_test_files();
+
+    assert!(
+        files.len() > 0,
+        "there are no test cases in the `instruction_counting` directory"
+    );
+
+    files.par_iter().for_each(|ent| {
+        let wasm_path = ent.path();
+        let do_not_instrument = false;
+        let module = wasm_test(&wasm_path, do_not_instrument).expect("can load module");
+
+        let region = MmapRegion::create(1, &Limits::default()).expect("region can be created");
+
+        let mut inst = region
+            .new_instance(module)
+            .expect("instance can be created");
+
+        inst.run("test_function", &[]).expect("instance runs");
+
+        let instruction_count = inst.get_instruction_count();
+        if let Some(_) = instruction_count {
```

Comment:
For situations where you are disregarding the inner value, (i.e. `Some(_)` or `Ok(_)`) there are also `is_some` and `is_none` methods! You can see more about `is_some` here: https://doc.rust-lang.org/std/option/enum.Option.html#method.is_some

So, this condition could instead look like:

```suggestion
        if instruction_count.is_some() {
```

Reply:
this is getting into _Real Pedantic Hours_™ however, so this isn't something I'd consider blocking by any means.

Reply:
OOPS

This is an outdated diff, you've already fixed this.